### PR TITLE
README: clean-up and clarifications

### PR DESCRIPTION
**Pull request by jonboulle**
- Remove redundant sentence describing media types document
- Disambiguate "OCI Layers", and more directly explain link to archival
  format
- Change ROADMAP pointer to a more realistic date
- Minor tweaks to FAQ wording

Signed-off-by: Jonathan Boulle <jonathanboulle@gmail.com>

### CODE REVIEW

**Comment by wking** [comment]
One nit that I don't feel strongly about [1], otherwise 4341c0808
looks good to me.

[1]: https://github.com/opencontainers/image-spec/pull/367/files#r81769571

File: README.md [hunk 1]
```diff
@@ -34,16 +34,16 @@ Once built the OCI Image can then be discovered by name, downloaded, verified by
...
+The [OCI Image Media Types](media-types.md) document is a starting point to understanding the overall structure of the specification.
 
-The high level components of the spec include:
+The high-level components of the spec include:
 
-* An [image manifest](manifest.md), a set of [filesystem layers](layer.md), and [image configuration](config.md) (base layer)
-* A process of hashing the image format for integrity and content-addressing (base layer)
-* Signatures that are based on signing image content address (optional layer)
-* Naming that is federated based on DNS and can be delegated (optional layer)
+* An archival format for container images, consisting of an [image manifest](manifest.md), an [image layout](image-layout.md), a set of [filesystem layers](layer.md), and [image configuration](config.md) (base OCI layer)
```
Comment (jonboulle) [inline README.md]: Eh, this sentence is already unwieldy - I'd rather let manifest.md be the canonical one and there are links to the manifest list from there

**Comment by vbatts** [comment]
LGTM

**Comment by philips** [comment]
LGTM

**Pull request merged**